### DISCUSSION TITLE
Filter entries by date

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9450] [Ideas] [answered]
> Created by: Ben Parizek (ben@barrelstrengthdesign.com) on 2015/03/12 23:19:32 +0000
> Votes at time of UserVoice import: 2

-----------------------------------

On some sites, it would be helpful to be able to filter Entries, Users, or Assets in a specific date range.

Additionally, several plugins may benefit from providing the option to filter entries by date and Users would benefit if the UI for that behavior was similar across all Element Types.

A simple implementation of this may just be to allow us to use a keyword syntax in the Search field: `from:2015-03-01`

A more advanced approach may allow us to use a Date filter designed in a similar way to the Status filter, or give us some option for an "Advanced Search" that provides a dropdown and extended UI for us to search for the specific content needed.

**Comment by brandonkelly** [comment, marked as answer]
This has been implemented for Craft 4 🎉  (#10075)
